pom.xml

```diff
@@ -250,6 +251,8 @@
         <HikariCP.version>4.0.3</HikariCP.version>
         <caffeine.version>2.9.3</caffeine.version>
         <kafka.clients.version>3.0.0</kafka.clients.version>
+        <!-- JDK1.8 only support google java format 1.7-->
+        <google.java.format.version>1.7</google.java.format.version>
```

Comment:
I can't find a version that says that jdk8 only supports google-java-format1.7 in the official documentation, can you provide the link?  thx